docs/tutorial/overview.md

```diff
@@ -136,9 +139,9 @@ par {
 }
 ```
 
-### Scheduling
-
-Currently, our compiler only supports multipication and addition operations between two values that are either integers or read from an array. To have more complex expressions, even something like `a + b + 5`, involves creation of multiple `group`s that will need to have their execution explicitly scheduled. We are currently in the process of deciding where scheduling should be handled: at the FuTIL level, or at the frontend level. Stay tuned!
-
+## Conclusion
 
-[astcode]: https://github.com/cucapra/futil/blob/mrxl/mrxl/mrxl/ast.py
```

Comment:
Looks like this reference got deleted. A bunch of other sentences still reference it.